test/methods/stream/publish-event.js

```diff
@@ -19,220 +19,214 @@ module.exports = () => {
 		describe("loginPrompt event", () => {
 			describe("this.userHasValidSession is truthy", () => {
 				it("calls displayNamePrompt", () => {
-					const stream = new Stream();
 					const displayNamePromptStub = sandbox.stub();
 
+					const stream = new Stream();
 					stream.userHasValidSession = true;
 					stream.displayNamePrompt = displayNamePromptStub;
 					stream.publishEvent({ name: 'loginPrompt' });
 
 					proclaim.isTrue(displayNamePromptStub.calledOnce);
-
 				});
 			});
 
 			describe("this.userHasValidSession is falsy", () => {
 				it("doesn't calls displayNamePrompt", () => {
-					const stream = new Stream();
 					const displayNamePromptStub = sandbox.stub();
 
+					const stream = new Stream();
 					stream.userHasValidSession = false;
 					stream.displayNamePrompt = displayNamePromptStub;
 					stream.publishEvent({ name: 'loginPrompt' });
 
 					proclaim.isFalse(displayNamePromptStub.calledOnce);
-
 				});
 			});
 		});
 
 		it("maps coral events to oComment events", (done) => {
```

Comment:
We're missing an assertion in this test, unless you meant to do that?

Reply:
This is intentional. In this test we are only expecting the event listener to be invoked. When this happens we call done to tell the framework the test is complete.

If the event listener is not invoked, we never call done and the test fails because of timeout.